### ISSUE TITLE
Does this work with ES6?

### ISSUE THREAD

**Issue by Ciantic** [s-panferov/awesome-typescript-loader#3]
I'm trying to use ES6, and it's import syntax but it's throwing weird errors.

I use now:

    loaders: [{
            test: /\.ts$/,
            loader: 'awesome-typescript-loader?target=es6&module=!ts-jsx-loader'
        }, ...]

I had to put `module=` since commonjs does not support es6 styled imports. Now I get: 

```
ERROR in ./index.ts
Cannot find global type 'Iterable'.

ERROR in ./index.ts
Cannot find global type 'Symbol'.

ERROR in ./index.ts
Cannot find global value 'Symbol'.

and the list goes on...
```

And my index.ts is basically empty:

```
/// <reference path='./index.d.ts' />
/// <reference path='./typings/jquery/jquery.d.ts' />
/// <reference path='./typings/react/react.d.ts' />
/// <reference path='./node_modules/ts-jsx-loader/react-jsx.d.ts' />
/// <reference path='./node_modules/typed-react/typed-react.d.ts' />

import run from "./Application";
run();
```

**Comment by s-panferov** [comment]
Thanks for your report, I'll check this today.

**Comment by s-panferov** [comment]
@Ciantic fixed in v0.2.2, please confirm.

**Comment by s-panferov** [comment]
@Ciantic 	feel free to reopen the issue if you have some troubles
